pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/filter/BaseFilter.java

```diff
@@ -68,8 +69,9 @@ protected void prepareNecessaryData(QueryParamsMap queryParamsMap) {
         } else if (savedFilters != null) {
             filters = savedFilters;
         }
-        this.from = Integer.parseInt(queryParamsMap.get(LIST_RANGE_FROM).value());
-        this.to = Integer.parseInt(queryParamsMap.get(LIST_RANGE_TO).value());
+        this.from = NumberUtils.toInt(queryParamsMap.get(LIST_RANGE_FROM).value(), 0);
+        this.to = NumberUtils.toInt(queryParamsMap.get(LIST_RANGE_TO).value(), 10000);
```

Comment:
I think it would be better to extract `0` and `10000` as constants with meaningful names to avoid so called magic numbers.